mypy.ini

```diff
@@ -13,7 +13,6 @@ disallow_incomplete_defs = true
 disable_error_code = empty-body
 # TODO: update our test error messages to match new mypy output
 show_error_codes = false
-force_uppercase_builtins = true
 force_union_syntax = true
```

Comment:
```suggestion
```

let's remove it as well.

Reply:
Agreed, I was planning to do it in a separate PR.